roles/harbor/templates/certificate.j2

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nginx-cert
+  namespace: harbor-system
+spec:
+  dnsNames:
+  - core.{{ domain }}
```

Comment:
Please standardize the indentation for list items in yaml.

Reply:
Fixed